src/SFA.DAS.EmployerAccounts.Web/Orchestrators/EmployerTeamOrchestrator.cs

```diff
@@ -220,6 +225,30 @@ public virtual async Task<OrchestratorResponse<AccountDashboardViewModel>> GetAc
                 var tasks = tasksResponse?.Tasks.Where(t => t.ItemsDueCount > 0 && t.Type != "AgreementToSign").ToList() ?? new List<AccountTask>();
                 var showWizard = userResponse.User.ShowWizard && userRoleResponse.UserRole == Role.Owner;
 
+
+                var apprenticeshipsCount = 0;
+                IEnumerable<GetApprenticeshipsResponse.ApprenticeshipDetailsResponse> apprenticeshipResponse = await GetApprenticeshipResponse(accountResponse);
+                if (apprenticeshipResponse != null)
+                {
+                    apprenticeshipsCount = apprenticeshipResponse.Count();
+                }
+
+                var cohortsCount = 0;
+                var draftApprenticeshipCount = 0;
+                CohortSummary singleCohort = new CohortSummary();
+                DraftApprenticeshipDto singleDraftApprenticeship = new DraftApprenticeshipDto();
+                if (apprenticeshipResponse == null)
+                {
+                    CohortSummary[] cohortsResponse = await GetCohortsResponse(accountResponse);
+                    if (cohortsResponse != null && cohortsResponse.Count() == 1)
+                    {
+                        cohortsCount = cohortsResponse.Count();
+                        singleCohort = cohortsResponse.First();
+                        draftApprenticeshipCount = singleCohort.NumberOfDraftApprentices;
+                        singleDraftApprenticeship = await GetSingleDraftApprenticeship(draftApprenticeshipCount, singleCohort, singleDraftApprenticeship);
+                    }
+                }

```

Comment:
Same as the other branch this doesn't belong here. Create a mediatr handler and supporting services the same as reservations and vacancies branch

Reply:
done